### PR TITLE
🗾 Display nearby resources on mapview list

### PR DESCRIPTION
**Pull request by ayanmallik**
## Description

- Pipe resources sorted by distance into list of mapview entries
- Enable selection of individual entry in list

Fixes #<number>

## Screenshots

image

### CODE REVIEW

File: client/src/components/MapViewList.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      }
+      setResources(newResources);
+    };
+
+    fetchResources();
+  }, []);
+
+  return (
+    <List
+      bordered
```
Comment (jackieo5023) [inline client/src/components/MapViewList.jsx:51]: I think we can probably get rid of bordered, unless this was intentional for the design

File: client/src/components/MapViewEntry.jsx [hunk 2]
```diff
@@ -11,40 +12,44 @@ type Props = {
...
-            <Col className="second">•</Col>
-            <Col className="third">1309 West Hill Street</Col>
+            <Col>{resource.city}</Col>
+            <Col className="second-element">•</Col>
+            <Col className="third-element">
+              {resource.address || 'Address not provided'}
+            </Col>
           </Row>
           <Row>
             <Col>$</Col>
```
Comment (jackieo5023) [inline client/src/components/MapViewEntry.jsx:33]: We can use the resource's cost for this!

File: client/src/components/MapViewEntry.jsx [hunk 3]
```diff
@@ -11,40 +12,44 @@ type Props = {
...
-            <Col className="second">•</Col>
-            <Col className="third">English, Spanish, French, Chinese</Col>
+            <Col className="second-element">•</Col>
+            <Col className="third-element">
+              {resource.languages.join(', ') || 'English'}
```
Comment (jackieo5023) [inline client/src/components/MapViewEntry.jsx:36]: I'm doing some super jank thing to show the languages in their respective language, can you do this here too 😂 https://github.com/hack4impact-uiuc/ymca/blob/0369e54d2fd3ffd72fd516b7fd65a5e391229abe/client/src/components/ResourcePreview.jsx#L72

**Review by jackieo5023 — APPROVED** [review]
Awesome!!! Excited for this to all come together. One general comment about CSS - CSS can sometimes be weird and conflict with each other (idk how that works), so in general I think it's good to preface your CSS names with maybe the component you're on - so like `mapview-second-element` or something like that

**Review by nkonjeti — APPROVED** [review]
Looks great Ayan!

**Pull request merged**